Netezos.Tests/Forging/LocalForgingTest.cs

```diff
@@ -76,5 +76,68 @@ public async Task CompareToFile()
                 }
             }
         }
+
+        [Fact]
+        public async Task CompareUnforgedFromFile()
```

Comment:
How I handled testing the unforge operations:

1. Read the JSON for baseline comparison.
2. Unforge the forged hex file bytes to a type.
3. JSON serialize the unforged type so it can be compared to the JSON baseline file.

Does that approach sound reasonable? Looking for more ideas.

Reply:
Alternative approach is compare `OperationContent` instanses using `Reflection`, instead of converting it to JSON and comparing `JsonElement`s field-by-field. But I think that's fine.